clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -465,7 +517,58 @@
     public static final Schema[] STOP_REPLICA_REQUEST = new Schema[] {STOP_REPLICA_REQUEST_V0};
     public static final Schema[] STOP_REPLICA_RESPONSE = new Schema[] {STOP_REPLICA_RESPONSE_V0};
 
-    /* an array of all requests and responses with all schema versions */
+    /* Update metadata api */
+
+    public static final Schema UPDATE_METADATA_REQUEST_PARTITION_STATE_V0 =
+            new Schema(new Field("topic", STRING, "Topic name."),
```

Comment:
OK, will do. I wasn't sure what was our policy when it came to referencing schemas from other APIs in the same class. Good to know.